### ISSUE TITLE
Feature request: support for webhooks

### ISSUE THREAD

**Issue by rob-3** [suneettipirneni/discord.js-embed-pagination#1]
As per [this](https://discord.com/developers/docs/interactions/receiving-and-responding#receiving-an-interaction), bots (especially serverless ones) can opt to receive interactions via webhook. Unfortunately, this means there is no `interaction` object exposed so I can't use this library. I'm curious if you have any ideas for a workaround

**Comment by suneettipirneni** [comment]
Are you using discord.js with your bot?

**Comment by rob-3** [comment]
Yes, https://github.com/rob-3/crumblbot is an example of how the webhook is used. See app.js
